style.css

```diff
@@ -162,15 +201,19 @@ nav > .mob-def {
   background-color: #1c1a19;
 }
 
-.flex-all {
-  height: 100%;
-  object-position: center;
+.flex-all,
+.intro {
   display: flex;
```

Comment:
- [ ] I would suggest you please fix the content of the paragraph as it is covering the whole width but if you see the Figma design then you will find it covers certain width. See the Image below for reference.
- [ ] Also use the same background color for buttons and hx tags and paragraphs.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213359712-0ef8a9dc-7618-496b-a69a-2de5ea134384.jpg)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/213359736-fca2cd00-a8b5-4222-ab05-e9a8b36c0e6b.jpg)